specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdateProfile.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "api-version": "2018-06-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/mySubscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines/myVMName/providers/Microsoft.GuestConfiguration/softwareUpdateProfile/default",
+        "name": "default",
+        "type": "Microsoft.GuestConfiguration/softwareUpdateProfile",
+        "properties": {
+          "computerName": "myVMName",
+          "computerIP": "40.85.217.32",
+          "computerEnvironment": "Azure",
+          "osType": "Windows",
+          "osName": "Microsoft Windows 10 Enterprise",
+          "osVersion": "10.0.17763 N/A Build 17763",
+          "osMajorVersion": "",
+          "osMinorVersion": "",
+          "osFullName": "",
+          "osArch": "x64",
+          "computerStartTime": "2018-10-30T17:31:02Z",
+          "restartPending": "True",
+          "guestConfigAgentVersion": "8.0.11081.0",
+          "guestConfigAgentStartTime": "2018-10-30T17:31:02Z",
+          "netRuntimeVersion": ".NET Framework 4.6.1",
+          "windowsUpdateSetting": "Notify before installation",
+          "windowsUpdateAgentVersion": "10.0.14393.2368",
+          "windowsUpdateIncludeRecommendedUpdates": "",
+          "wsusServer": "http://rd2-sup-hlb.redmond.corp.microsoft.com:80",
+          "linuxPackageManager": "",
+          "criticalCount": "2",
+          "securityCount": "0",
+          "otherCount": "0",
+          "assessmentRuntime": "2018-10-30T17:31:02Z",
+          "oldestMissingSecurityUpdateInDays": "35",
+          "oldestMissingSecurityUpdateBucket": "30 days ago",
+          "lastUpdateApplied": "",
```

Comment:
"" is not a valid value for a `date-time` property

Reply:
Removed date-time format for this property because this can actually not be a valid date-time in the when updates have never been applied to VM.